src/server/decommission-service/controllers/decommission-form.js

```diff
@@ -0,0 +1,27 @@
+import { buildOptions } from '~/src/server/common/helpers/options/build-options.js'
```

Comment:
So the folders reflect the urls on the frontend. Looks like this is an admin section. So please put this folder in the `admin` folder and update the urls to be in the admin section